pkg/services/folder/folderimpl/sqlstore_test.go

```diff
@@ -183,6 +196,113 @@ func TestIntegrationDelete(t *testing.T) {
 	})
 }
 
+func TestIntegrationDeleteNestedFolders(t *testing.T) {
```

Comment:
This file contains only tests for the nested folder store. Therefore I would suggest to move this to the folder service tests (since we test the folder service Delete() behaviour).